plugin/src/main/kotlin/com/jraska/module/graph/assertion/ModuleGraphAssertionsPlugin.kt

```diff
@@ -29,13 +29,11 @@ class ModuleGraphAssertionsPlugin : Plugin<Project> {
   override fun apply(project: Project) {
     val graphRules = project.extensions.create(GraphRulesExtension::class.java, Api.EXTENSION_ROOT, GraphRulesExtension::class.java)
 
-    project.afterEvaluate {
```

Comment:
Because of this (correct) change, tests in `ModuleGraphAssertionsPluginTest` started to fail. It is rather problem with the tests depending on implementation detail 🤦 

I deleted and modified these faulty tests here: https://github.com/jraska/modules-graph-assert/pull/273. This change is good to go.